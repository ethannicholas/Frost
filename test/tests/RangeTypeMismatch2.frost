class RangeTypeMismatch2 {
    @class
    method main() {
        for i in 255 .. "Hello" by -1 {
            Console.printLine(i)
        }
    }
}