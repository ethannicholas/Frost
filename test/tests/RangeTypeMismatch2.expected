RangeTypeMismatch2.panda:4:22: error: range endpoints must be value types, but found 'panda.core.Immutable'
1 error
